js/apps/system/_admin/aardvark/APP/react/package.json

```diff
@@ -27,7 +27,7 @@
     "ajv": "^8.12.0",
     "ajv-errors": "^3.0.0",
     "ansi-html-community": "0.0.8",
-    "arangojs": "^8.7.0",
+    "arangojs": "^9.0.0-preview.3",
```

Comment:
let's split the version update into another PR as discussed, since we are not directly dependent on it here